examples/systemcalls/systemcalls.c

```diff
@@ -16,7 +23,9 @@ bool do_system(const char *cmd)
  *   and return a boolean true if the system() call completed with success 
  *   or false() if it returned a failure
 */
-
+    int ret = system(cmd);
+    if(ret)
```

Comment:
@lnxblog think about what would happen if the cmd returned with an error code -1 ?

Reply:
if the return was -1 would the if condition fail?